Yona/Yona/Others/SMSValidPasscodeLogin/LoginSignupValidationMasterView.swift

```diff
@@ -157,7 +157,7 @@ extension LoginSignupValidationMasterView {
                     if success {
                         
                         setViewControllerToDisplay(ViewControllerTypeString.pinResetValidation, key: YonaConstants.nsUserDefaultsKeys.screenToDisplay)
-                        //self.performSegue(withIdentifier:R.segue.loginViewController.transToPinResetValidation, sender: self)
+                       // self.performSegue(withIdentifier:R.segue.loginViewController.transToPinResetValidation, sender: self)
```

Comment:
Remove lines you don't need, instead of commenting them. You can always find it back in Git.